candi/openapi/cluster_configuration.yaml

```diff
@@ -127,6 +127,8 @@ apiVersions:
         type: object
         description: |
           Global proxy setup (especially for working in air-gapped environments).
+          
+          The parameters described in this section will be translated into the environment variables `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` for all cluster nodes and Deckhouse components. This will lead to HTTP(S) requests (curl, git, registry, etc.) to all resources not specified in the `noProxy` parameter being executed through a proxy. The `podSubnetCIDR`, `serviceSubnetCIDR` subnets, and the `clusterDomain` domain are added to `noProxy` automatically.
```

Comment:
```suggestion
          The parameters described in this section will be translated into the environment variables `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` for all cluster nodes and Deckhouse components. This will result in HTTP(S) requests (curl, git, registry, etc.) to all resources not listed in the `noProxy` parameter being made through a proxy. Note that the `podSubnetCIDR` and `serviceSubnetCIDR` subnets, as well as the `clusterDomain` domain are added to `noProxy` automatically.
```